pages/publish-nft-book/index.vue

```diff
@@ -300,4 +301,13 @@ const handleIscnInput = async () => {
   }
 }
 
+const showErrorToast = (message: string) => {
+  toast.add({
+    icon: 'i-heroicons-exclamation-circle',
+    title: message,
+    timeout: 3000,
+    color: 'red'
+  })
+}
```

Comment:
Should this be a composable so that we can use it everywhere?

Reply:
[♻️ Refactor toast error to use composable](https://github.com/likecoin/nft-book-press/pull/352/commits/be2f3f586252ec5bc23f2a1d92c09228506e7154)